### PR TITLE
Added documentation for refreshing local documentation changes

### PR DESCRIPTION
**Pull request by amogh-jrules**
Solves #3910

### CODE REVIEW

**Comment by amogh-jrules** [comment]
Can someone review this?

**Comment by nabobalis** [comment]
Sorry @amogh-jrules, I had a stab at reworking more of the text since it was being worked on.

File: docs/dev_guide/documentation.rst [hunk 1]
```diff
@@ -114,10 +113,16 @@ This will generate HTML documentation for SunPy in the "docs/_build/html" direct
 You can open the "index.html" file to browse the final product.
 The gallery examples are located under "docs/_build/html/generated/gallery".
 Sphinx builds documentation iteratively, only adding things that have changed.
-If you'd like to start from scratch then just run::
+
+If you'd like to start from scratch (i.e., remove the tox cache) then just run::
 
     $ tox -e build_docs -- -aE
 
+To build the documentation locally you must have all the dependencies (``pip install -e .[docs]``) specified in ``setup.cfg`` installed on your computer.
```
Comment (dstansby) [inline docs/dev_guide/documentation.rst]:
```suggestion
To build the documentation in your current python environment you must have all the dependencies specified in ``setup.cfg`` installed (``pip install -e .[docs]``).
```

File: docs/dev_guide/documentation.rst [hunk 2]
```diff
@@ -114,10 +113,16 @@ This will generate HTML documentation for SunPy in the "docs/_build/html" direct
...
+Then go into to the"docs" directory and run::
```
Comment (dstansby) [inline docs/dev_guide/documentation.rst]:
```suggestion
Then change to the :file:`docs/` directory and run::
```

**Comment by dstansby** [comment]
Thanks for the PR! I added a couple of suggested changes above, but nothing major.

File: docs/dev_guide/documentation.rst [hunk 3]
```diff
@@ -114,10 +113,16 @@ This will generate HTML documentation for SunPy in the "docs/_build/html" direct
 You can open the "index.html" file to browse the final product.
 The gallery examples are located under "docs/_build/html/generated/gallery".
 Sphinx builds documentation iteratively, only adding things that have changed.
-If you'd like to start from scratch then just run::
+
+If you'd like to start from scratch (i.e., remove the tox cache) then just run::
```
Comment (dpshelio) [inline docs/dev_guide/documentation.rst]:
```suggestion
If you'd like to start from scratch (i.e., remove the tox cache) then run::
```
"just" doesn't add much to it and makes it look as it's easy to understand what's going on.

**Pull request merged**